### PR TITLE
Document missing points for 1.3

### PR DESCRIPTION
**Pull request by XVincentX**
- Document the (small) schema API
- Illustrate the new `-f` flag for credentials list
- Show the Typescript support

### CODE REVIEW

File: docs/admin/index.md [hunk 1]
```diff
@@ -4,9 +4,9 @@ title: Admin API Reference
 doc-order: 10.0
 ---
 
-### Overview 
+### Overview
 
-The Express Gateway Admin API is a REST interface to manage internal entities such as Users, Applications, Credenitials, Scopes, OAuth2 tokens and the relationships among such entities.
```
Comment (DrMegavolt) [inline docs/admin/index.md]: whats wrong with word REST ?

File: docs/admin/schemas/index.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+the gateway from starting if some required parameters are missing or incorrect. You can query the schemas through the
+API and provide new ones using [plugins][plugins]
+
+Express Gateway ships with some built-in schemas for [policies][policies], [conditions][conditions] — [plugins][plugins]
+can eventually define other personal schemas to validate the provided options.
+
+### Get All Schemas
+##### Request: `GET /schemas/:type/:name`
+
+With `type: condition` and `name: oneOf`
```
Comment (DrMegavolt) [inline docs/admin/schemas/index.md]: can you add example with full path `/schema/condition/oneOf`

File: docs/cli/credentials/list.md [hunk 3]
```diff
@@ -15,17 +15,19 @@ eg credentials list -c <consumerId> [options]
...
+| `-f, --filter`     | `array`    | `active` | Shows `active`, `archived` credentials, or both |
+
 
 ### Examples
 
 #### List all credentials of consumer
 
 ```shell
-$ eg credentials list -c oauth2 2e90e145-bc4b-4ff1-a63a-a097644ba360
+$ eg credentials list -c 2e90e145-bc4b-4ff1-a63a-a097644ba360 -f active archived
```
Comment (DrMegavolt) [inline docs/cli/credentials/list.md]: will it work with username instead of id?

File: docs/admin/index.md (hunk 1 above)
Comment (kevinswiber) [reply docs/admin/index.md]: Long story...  :smile:

Comment (kevinswiber) [reply docs/admin/index.md]: An HTTP interface?  A REST*ful* interface?

Comment (XVincentX) [reply docs/admin/index.md]: I'll go with `HTTP`

File: docs/admin/schemas/index.md (hunk 2 above)
Comment (XVincentX) [reply docs/admin/schemas/index.md]: Yeah sure, probably it's way more clear.

**Pull request merged**